pages/Guides/supernode-program/docs.en.md

```diff
@@ -8,90 +8,89 @@ taxonomy:
+  Opening 7890 allows your node to be a full node and contribute to the network. If you were successful you will see your node's name appear on [Explorer](https://explorer.nemtool.com/#/nodelist). This might take a while to appear.
 
   Opening port 7880 is required to allow the servant to work correctly.
 
   Port 7778 allows light wallets to connect to the server.
 
-- **Step 9**: Start NIS, let it synchronize and then start the servant.
-
-  To start the servant double click on ``runservant.bat`` in the ``servant`` folder (In Mac or Linux navigate to the file in terminal and use the command ``sh startservant.sh``). If it works you will see your terminal/command prompt window become active.
-
-- **Step 10**: Send an unencrypted enrollment message to the official node rewards account.
+- **Step 10**: Start NIS, let it synchronize, and then start the servant.
 
-  Send a NEM message to ``NAFUNDBUKIOSTMD4BNXL7ZFE735QHN7A3FBS6CMY`` stating "enroll (your IP or hostname) (your nodes name) (your delegated harvesting public key)" from the primary account that has the 3 million XEM deposit to officially announce you would like to participate in the program. To find the delegated public key, open your 3 million XEM account in NCC. Go to *View Account Details* and then you can see the delegated account's public key. Please note that this is **not** the delegated harvesting **private** key you used earlier.
+  To start the servant double-click on ``runservant.bat`` in the ``servant`` folder. (In Mac or Linux navigate to the file in terminal and use the command ``sh startservant.sh``. It's recommended to run in background using ``screen`` or ``nohup`` command.)
 
-  ![find delegated public key](delegated-key.png)
+- **Step 11**: Enroll in the Program using NEM Wallet.
 
-  ![Example for an enrollment message](enroll-1.png)
+  Go to Services -> SuperNode Program -> Check & Enroll in Program -> Enroll in Program, put the enrollment address for current month (announced via official Twitter and Discord) and your node host for your node. Send enrollment transaction.
+  Make sure host matches your node host from ``/node/info`` endpoint response.
 
-- **Step 11**: Review your results at [supernodes.nem.io](http://supernodes.nem.io/) to make sure that your node is passing all the tests. Please be patient, since your results will not be visible immediately.
+  ![Endpoint response](node-info.png)
 
-  For more discussion about this blog, please visit the [NEM Forum](https://web.archive.org/web/20210814074405/https://forum.nem.io/t/nem-supernode-rewards-program/1735).
+  ![NEM Wallet Enrollment](wallet-enroll.png)
 
-## Editing Your Supernode IP and Alias
+  To breathe life into NEM, the new program will require monthly re-enrollments in order to increase engagements.
```

Comment:
Change this to:

'The Supernode Program requires monthly re-enrollments. We believe the set-and-forget philosophy of node operation rewards unhealthy behaviours, especially in a decentralised network where validators need to make informed decisions pertaining to the state of the network.'